inc/commondbtm.class.php

```diff
@@ -5273,4 +5273,43 @@ private function assetBusinessRules($condition) {
          }
       }
    }
+
+   /**
+    * Correct entity id if needed when cloning a template
+    *
+    * @param array  $data
+    * @param string $parent_field
+    *
+    * @return array
+    */
+   public static function checkTemplateEntity(array $data, $parent_field) {
```

Comment:
It may be less complicated and more secure to directly have `$parent_id` and `$parent_itemtype` params.

Reply:
Done in a75c63a.

Reply:
and 42c58c1.